src/UserInfo.css

```diff
@@ -0,0 +1,22 @@
+.userInfoSection {
+    margin-bottom: 20px;
```

Comment:
nit: it's generally better to use rem instead of px

Reply:
Git it got that fixed.